index.php

```diff
@@ -42,6 +59,16 @@
             . '</div> ';
     }
     echo $html;
+
+    $query = $pdo->prepare('SELECT `shows`.`name`, `genres`.`name` AS `genre`, `release year` FROM `shows` INNER JOIN `genres` ON `shows`.`genre` = `genres`.`id`;');
+
+    $query->execute();
+
+    $result = $query->fetchALL();
```

Comment:
Is $result meant to be $items?

Reply:
yeah i was testing to see if i could get the database items to show up

Reply:
Okay sounds good, approved!